api/controllers/services/scosv2/locationService.js

```diff
@@ -0,0 +1,80 @@
+const Location = require("../../models/location")
+const AbstractService = require('../abstractService')
+const latLong = require('../csv/latLong')
+const QueryUtils = require('../../../utils/query')
+
+class LocationService extends AbstractService {
+
+    constructor() {
+        super("agency_location")
+        this.agencyLocationResourceId = this.config.agency_location_resource
+        this.serviceLocationResourceId = this.config.service_location_resource
+        this.uri = `${this.host}/api/v1/organization/handson_central_ohio/dataset/location_consolidated/query?_format=json`
+
+        this.queryFields = `
```

Comment:
is there a reason we are just returning specific columns? Should we return it all so the UI or whatever is calling this has access to everything? I am fine either way, just figured i would bring it up.

Reply:
actually this is a dead code, using vscode now that has eslint built in to alert.